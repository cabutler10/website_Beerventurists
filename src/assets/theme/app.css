html {
  height: 100%;
}

body {
  margin: 0;
  padding: 0;
  font-family: 'Open Sans', sans-serif;
  background-color: #ffffff;
  overflow-x: hidden;
  font-size: 14px;
  width: '100%';
}

a {
  text-decoration: none;
}
