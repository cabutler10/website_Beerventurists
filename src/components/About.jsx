import React from 'react'
import PropTypes from 'prop-types'
import { useTranslation } from 'react-i18next'
import { makeStyles } from '@material-ui/styles'
import Typography from '@material-ui/core/Typography'

const useStyles = makeStyles(theme => ({
  root: {
    paddingTop: 30,
    paddingBottom: 30,
    paddingLeft: 15,
    paddingRight: 15,
    textAlign: 'center',
    [theme.breakpoints.up('sm')]: {
      paddingLeft: 60,
      paddingRight: 60,
    },
    [theme.breakpoints.up('md')]: {
      paddingTop: 60,
      paddingBottom: 60,
      paddingLeft: 90,
      paddingRight: 90,
    },
    [theme.breakpoints.up('lg')]: {
      paddingTop: 45,
      paddingBottom: 90,
      paddingLeft: 170,
      paddingRight: 170,
    },
    [theme.breakpoints.up('xl')]: {
      paddingTop: 90,
      paddingBottom: 90,
      paddingLeft: 400,
      paddingRight: 400,
    },
  },
  title: {
    textTransform: 'uppercase',
    fontWeight: 700,
    fontSize: 16,
    [theme.breakpoints.up('md')]: {
      fontSize: 36,
    },
  },
  text: {
    paddingTop: 30,
    color: theme.palette.common.black,
    fontSize: 12,
    [theme.breakpoints.up('md')]: {
      fontSize: 14,
    },
  },
}))

function About() {
  const classes = useStyles()
  const { t } = useTranslation()
  return (
    <div className={classes.root} id="about">
      <Typography className={classes.title} variant="h5">
        {t('about.sectionTitle')}
      </Typography>
      <Typography className={classes.text} variant="body1">
        {t('about.message')}
      </Typography>
    </div>
  )
}

About.propTypes = {
  img: PropTypes.object.isRequired,
  waveImg: PropTypes.object.isRequired,
}

export default About
