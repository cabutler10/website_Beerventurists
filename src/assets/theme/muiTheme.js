import { createMuiTheme } from '@material-ui/core/styles'
import 'typeface-open-sans'
import 'typeface-permanent-marker'
import './app.css'

const status = {
  white: '#ffffff',
  black: '#040001',
  greyLt: '#F7F5F5',
  greyMed: '#656565',
  grey: '#c5c6c7',
  greyDk: '#404040',
  green: '#00942b',
  yellow: '#fce566',
  red: '#FF0012',
  blue: '#015b99',
  blueLt: '#0172c0',
  blueDk: '#014472',
  orange: '#c56113',
  orangeDk: '#c56113',
  orangeLt: '#ebab2f',
  tan: '#FFFBEF',
}

const theme = createMuiTheme({
  palette: {
    type: 'light',
    primary: {
      light: status.blueLt,
      main: status.blue,
      dark: status.blueDk,
    },
    secondary: {
      light: status.orangeLt,
      main: status.orange,
      dark: status.orangeDk,
    },
    error: { main: status.red },
  },
  status,
  typography: {
    useNextVariants: true,
    fontFamily: "'Open Sans', sans-serif",
    h5: {
      letterSpacing: 3,
    },
    h2: {
      fontFamily: "'Permanent Marker', cursive",
    },
    body1: {
      letterSpacing: 1,
      lineHeight: 1.8,
      //TODO is this an error in MUI? not sure why its default is 16 when MUI default is 14
      fontSize: 14,
    },
    body2: {
      letterSpacing: 1,
      fontSize: 14,
    },
    button: {
      letterSpacing: 3,
    },
  },
  overrides: {
    MuiTypography: {
      colorPrimary: {
        color: status.black,
      },
      colorSecondary: {
        color: status.orangeLt,
      },
      colorTextSecondary: {
        color: status.white,
      },
    },
    MuiButton: {
      root: {
        padding: '9px 27px',
        borderRadius: 0,
      },
      contained: {
        boxShadow: 'none',
      },
    },
  },
})

export default theme
