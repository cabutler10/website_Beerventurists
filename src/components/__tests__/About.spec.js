import React from 'react'
import { render } from 'react-testing-library'
import About from '../About'
import { ThemeProvider } from '@material-ui/styles'
import theme from '../../assets/theme/muiTheme'

describe('About', () => {
  it('renders correctly', () => {
    const mockProps = {
      img: {
        node: {
          fluid: {
            aspectRatio: 0,
            src: 'mockString',
            srcSet: 'mockString',
            sizes: 'mockString',
          },
        },
      },
      waveImg: {
        node: {
          fluid: {
            aspectRatio: 0,
            src: 'mockString',
            srcSet: 'mockString',
            sizes: 'mockString',
          },
        },
      },
      t: () => '',
    }
    const component = render(
      <ThemeProvider theme={theme}>
        <About {...mockProps} />
      </ThemeProvider>
    )
    expect(component).toMatchSnapshot()
  })
})
