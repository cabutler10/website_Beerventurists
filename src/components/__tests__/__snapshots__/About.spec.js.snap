// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About renders correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="Hook-root-3rw14z"
        id="about"
      >
        <div
          class="Hook-textContainer-1eodug1"
        >
          <h5
            class="MuiTypography-root-1 MuiTypography-h5-17 Hook-title-62s0fu"
          >
            about.sectionTitle1
          </h5>
          <h5
            class="MuiTypography-root-1 MuiTypography-h5-17 Hook-title-62s0fu"
          >
            about.sectionTitle2
          </h5>
          <p
            class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-mjxm1l"
          >
            about.message
          </p>
        </div>
        <div
          class="Hook-imgContainer-z1deon"
        >
          <div
            class="Hook-img-1vffzme gatsby-image-wrapper"
            style="position: relative; overflow: hidden;"
          >
            <div
              style="width: 100%;"
            />
            <picture>
              <img
                alt="two surfers in ocean"
                sizes="mockString"
                src="mockString"
                srcset="mockString"
                style="position: absolute; top: 0px; left: 0px; width: 100%; height: 100%; opacity: 0;"
              />
            </picture>
            <noscript>
              &lt;picture&gt;&lt;img sizes="mockString" srcset="mockString" src="mockString" alt="two surfers in ocean" style="position:absolute;top:0;left:0;opacity:1;width:100%;height:100%;object-fit:cover;object-position:center"/&gt;&lt;/picture&gt;
            </noscript>
          </div>
        </div>
        <div
          class="Hook-backgroundImg-1kaykh3 gatsby-image-wrapper"
          style="position: absolute; overflow: hidden;"
        >
          <div
            style="width: 100%;"
          />
          <picture>
            <img
              alt="wave background"
              sizes="mockString"
              src="mockString"
              srcset="mockString"
              style="position: absolute; top: 0px; left: 0px; width: 100%; height: 100%; opacity: 0;"
            />
          </picture>
          <noscript>
            &lt;picture&gt;&lt;img sizes="mockString" srcset="mockString" src="mockString" alt="wave background" style="position:absolute;top:0;left:0;opacity:1;width:100%;height:100%;object-fit:cover;object-position:center"/&gt;&lt;/picture&gt;
          </noscript>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="Hook-root-3rw14z"
      id="about"
    >
      <div
        class="Hook-textContainer-1eodug1"
      >
        <h5
          class="MuiTypography-root-1 MuiTypography-h5-17 Hook-title-62s0fu"
        >
          about.sectionTitle1
        </h5>
        <h5
          class="MuiTypography-root-1 MuiTypography-h5-17 Hook-title-62s0fu"
        >
          about.sectionTitle2
        </h5>
        <p
          class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-mjxm1l"
        >
          about.message
        </p>
      </div>
      <div
        class="Hook-imgContainer-z1deon"
      >
        <div
          class="Hook-img-1vffzme gatsby-image-wrapper"
          style="position: relative; overflow: hidden;"
        >
          <div
            style="width: 100%;"
          />
          <picture>
            <img
              alt="two surfers in ocean"
              sizes="mockString"
              src="mockString"
              srcset="mockString"
              style="position: absolute; top: 0px; left: 0px; width: 100%; height: 100%; opacity: 0;"
            />
          </picture>
          <noscript>
            &lt;picture&gt;&lt;img sizes="mockString" srcset="mockString" src="mockString" alt="two surfers in ocean" style="position:absolute;top:0;left:0;opacity:1;width:100%;height:100%;object-fit:cover;object-position:center"/&gt;&lt;/picture&gt;
          </noscript>
        </div>
      </div>
      <div
        class="Hook-backgroundImg-1kaykh3 gatsby-image-wrapper"
        style="position: absolute; overflow: hidden;"
      >
        <div
          style="width: 100%;"
        />
        <picture>
          <img
            alt="wave background"
            sizes="mockString"
            src="mockString"
            srcset="mockString"
            style="position: absolute; top: 0px; left: 0px; width: 100%; height: 100%; opacity: 0;"
          />
        </picture>
        <noscript>
          &lt;picture&gt;&lt;img sizes="mockString" srcset="mockString" src="mockString" alt="wave background" style="position:absolute;top:0;left:0;opacity:1;width:100%;height:100%;object-fit:cover;object-position:center"/&gt;&lt;/picture&gt;
        </noscript>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllBySelectText": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getAllByValue": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getBySelectText": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "getByValue": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllBySelectText": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryAllByValue": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryBySelectText": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "queryByValue": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
